vnext/overrides.json

```diff
@@ -548,6 +548,42 @@
       "type": "platform",
       "file": "src/RNTester/js/examples-win/Keyboard/KeyboardFocusExample.windows.tsx"
     },
+    {
+      "type": "platform",
+      "file": "src/RNTester/js/examples-win/LegacyTests/AccessibilityTestPage.tsx"
```

Comment:
If you want, you could also add "examples-win" to the list of unenforced directories at the top of the file and remove the overrides listed for it. Nothing in that directory should need three-way-merge, so it seems safe.

Reply:
done

---
In reply to: [457676898](https://github.com/microsoft/react-native-windows/pull/5555#discussion_r457676898) [](ancestors = 457676898)